### PR TITLE
Add Week 3,4,5 updates

### PR DESCRIPTION
**Pull request by anoura12**
Pushing Week 3, 4 and 5 GSoC Updates

### CODE REVIEW

File: applications/GSoC/AR-weekly-updates/week4.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+**Planned**
+
+* Present MVP to the community
+* Add documentation for the code
+* List feedback from the community and make changes
+
+**Blockers**
+
+Missed out on a bit of time due to side effects from the vaccine. Will be covering up remaining work this weekend.
```
Comment (KirstieJane) [inline applications/GSoC/AR-weekly-updates/week4.md:16]: I'm glad you got the vaccine @anoura12!! Don't worry about catching up after illness! 💉 ❤️

File: applications/GSoC/AR-weekly-updates/week5.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Week 5 (5th July - 11th July)
+
+**Done**
+
+* PRs #114 and #115 got merged.
```
Comment (KirstieJane) [inline applications/GSoC/AR-weekly-updates/week5.md:5]: Can you make these links to the PRs @anoura12 ?

**Review by KirstieJane — APPROVED** [review]
Looks great - thank you @anoura12 !

I've left a small comment - if you can make those PRs into links that would be great. No need for me to re-review, go ahead and merge when that's done. 

Thank you for all the great work!

File: applications/GSoC/AR-weekly-updates/week5.md (hunk 2 above)
Comment (anoura12) [reply applications/GSoC/AR-weekly-updates/week5.md:5]: Oh yes sure! I forgot to do that here.

File: applications/GSoC/AR-weekly-updates/week4.md (hunk 1 above)
Comment (anoura12) [reply applications/GSoC/AR-weekly-updates/week4.md:16]: Thank you so much @KirstieJane !

**Pull request merged**